### PR TITLE
NER version switched to 2, and fixing custom with no domain set

### PR DESCRIPTION
**Pull request by anneCarlson**
Reviewers: 
----------

- [x] @madisonmay 
- [ ] @anneCarlson

What's the purpose of the PR?
-----------------------------

Updates version number of ner.
https://trello.com/c/voWeqH80/1986-3-ner-updated-in-client-libs-default-version
Fix for custom domain is in there as well.

Description of changes
----------------------

Notes to reviewers
------------------

Keep in mind while reviewing code:
----------------------------------
- Is relevant code tested?
- Are added functions/methods documented?
- Separation of concerns (SOC)
- Don't repeat yourself (DRY) 
- Limit the number of positional arguments
- Is function/method length reasonable?
- Can code be broken down into smaller components?
- Where do added functions/methods belong?
- Are variable names descriptive?
- Are errors handled appropriately?
- Can logic be simplified?
- Does the code make sense in context? (expand the diff)

### CODE REVIEW

**Comment by sihrc** [comment]
Tests for v1? (Also, seems like tests aren't passing)

File: IndicoIo/IndicoIo.php [hunk 1]
```diff
@@ -90,7 +90,7 @@ public static function fer($image, $params=array())
 
 	public static function keywords($text, $params=array())
 	{
-		if (!array_key_exists('v', $params) || !array_key_exists('version', $params)) {
```
Comment (madisonmay) [inline IndicoIo/IndicoIo.php]: Great catch

File: IndicoIo/IndicoIo.php [hunk 2]
```diff
@@ -109,16 +109,25 @@ public static function twitter_engagement($text, $params=array())
 
     public static function people($text, $params=array())
     {
+        if (!array_key_exists('v', $params) && !array_key_exists('version', $params)){
```
Comment (madisonmay) [inline IndicoIo/IndicoIo.php]: This is kind of gross code and we use it a lot.  Could we factor it out into a utility function that we use?

**Comment by madisonmay** [comment]
One minor comment, otherwise, LGTM

**Pull request merged**